runelite-client/src/main/java/net/runelite/client/plugins/wikilookup/WikiLookupConfig.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2018, Tanner Mjelde
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package net.runelite.client.plugins.wikilookup;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("wikilookup")
+public interface WikiLookupConfig extends Config
+{
+	@ConfigItem(
+			position = 0,
+			keyName = "boundToShift",
+			name = "Shift bound lookup",
+			description = "Determines if wiki lookup will enable while shift is held."
+	)
+	default boolean boundToShift()
+	{
+		return true;
+	}
+	
+	@ConfigItem(
+			position = 1,
+			keyName = "removeNonVital",
+			name = "Make wiki option default",
+			description = "Determines whether wiki is the only option available during lookup."
+	)
+	default boolean removeNonVital()
+	{
+		return false;
+	}
+	
+	@ConfigItem(
+			position = 2,
+			keyName = "permWiki",
+			name = "Make wiki option always visible",
+			description = "Makes the wiki option always visible but never the default regardless of options."
+	)
+	default boolean permWiki()
```

Comment:
How is this useful? Probs not necessary

Reply:
This is the option that simply overrides everything and makes it so that anywhere in game the wiki option is always visible but not the default....kind of like examine. Personally this is what I have been using.